test/filters/test_sobel.py

```diff
@@ -130,7 +130,9 @@ def test_noncontiguous(self, device, dtype):
         inp = torch.rand(3, 5, 5, device=device, dtype=dtype).expand(batch_size, -1, -1, -1)
 
         actual = kornia.filters.spatial_gradient(inp)
-        assert_allclose(actual, actual)
+
+        assert inp.is_contiguous() is False
```

Comment:
I would test also that the result is contiguous

Reply:
Done